src/Raven.Server/Documents/Indexes/MapIndexBase.cs

```diff
@@ -32,10 +32,11 @@ protected override IIndexingWork[] CreateIndexWorkExecutors()
 
         public override IDisposable InitializeIndexingWork(TransactionOperationContext indexContext)
         {
-            if (sizeof(int) == IntPtr.Size || DocumentDatabase.Configuration.Storage.ForceUsing32BitsPager)
-                return null;
+            var mode = sizeof(int) == IntPtr.Size || DocumentDatabase.Configuration.Storage.ForceUsing32BitsPager
```

Comment:
This is a 'desired' mode, but if you are loading the bloom filters the first one is calculating the actual mode: https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Documents/Indexes/CollectionOfBloomFilters.cs#L112